src/evm.ts

```diff
@@ -40,3 +41,17 @@ export const decodeRevertString = (returndata: string) => {
 
   return goDecode.data;
 };
+
+export const estimateMulticallGasLimit = async (
+  contract: SelfMulticall,
+  calldatas: string[],
+  fallbackGasLimit: ethers.BigNumber | undefined
+) => {
+  const goEstimateGas = await go(async () => contract.estimateGas.multicall(calldatas));
+  if (goEstimateGas.success) {
+    // Adding a extra 10% because multicall consumes less gas than tryMulticall
+    return goEstimateGas.data.mul(ethers.BigNumber.from(Math.round(1.1 * 100))).div(ethers.BigNumber.from(100));
+  }
+
+  return fallbackGasLimit ?? ethers.BigNumber.from(2_000_000);
```

Comment:
As a note, in Airseeker v2 we only [rely on the fallback gas limit and error out if it's not provided](https://github.com/api3dao/airseeker-v2/blob/f15339a9dfb37a4b4a1897862e22dff9f57d8871/src/update-feeds-loops/submit-transactions.ts#L170) and RPC gas estimation fails.

Reply:
Right, I thought we might as well try using a default gas limit value and try to fund the recipients using that but I'm aware this is a magic number that might not be a good fit on some chains. It is used as a the last option because I'm thinking that we might remove the `fulfillmentGasLimit` config option from the [config generator script](https://github.com/api3dao/manager-multisig/issues/344). If that is true then we won't have a fallback value to use in the `estimateMulticalGasLimit` function. Ideas/suggestions are always welcome